packages/tablecloth-native/tablecloth-native.0.0.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis:
+  "Native OCaml library implementing Tablecloth, a cross-platform standard library for OCaml, Bucklescript and ReasonML"
+description: "Longer description"
+maintainer: "Paul Biggar <paul@darklang.com>"
+authors: "Paul Biggar <paul@darklang.com>"
+license: "MIT with some exceptions"
+homepage: "https://github.com/darklang/tablecloth"
+bug-reports: "https://github.com/darklang/tablecloth/issues"
+depends: [
+  "ocaml"
+  "ocamlfind"
```

Comment:
This dependency doesn't seem to be used anywhere in your source code. Could you remove it?

Reply:
👍